src/shared/components/radio-group/RadioGroup.scss

```diff
@@ -28,18 +28,33 @@
       float: left;
       margin-top: 2px;
       margin-left: 2px;
-      width: 9px;
-      height: 9px;
+      height: var(--radio-checked-diameter, 7px);
+      width: var(--radio-checked-diameter, 7px);
       border-radius: 50%;
       background: $green;
     }
   }
 }
 
 .label {
+  color: var(--radio-label-color, #{$black});
+  font-size: var(--radio-label-font-size, 12px);
+  font-weight: var(--radio-label-font-weight, #{$light});
   line-height: 30px;
-  font-size: 14px;
   padding-right: 2px;
   word-wrap: break-word;
   cursor: pointer;
 }
+
+.themeLight {
+  --radio-label-font-size: 13px;
```

Comment:
Same as the comment in the checkbox scss.

Reply:
✅